packages/appstore/src/components/onboarding-new/static-dashboard.tsx

```diff
@@ -0,0 +1,502 @@
+                                                                        is_blurry.options_description,
+                                                                }
+                                                            )}
+                                                        />,
+                                                    ]}
+                                                />
+                                            )}
+                                        </Text>
+                                    </div>
+                                )}
+                            </div>
+                            {has_account && (
+                                <StaticCurrencySwitcherContainer
+                                    title={
+                                        <Text size='xs' line_height='s'>
+                                            {is_eu ? localize(`EUR`) : localize(`US Dollar`)}
+                                        </Text>
+                                    }
+                                    icon={is_eu ? 'EUR' : 'USD'}
+                                >
+                                    <BalanceText currency={is_eu ? 'EUR' : 'USD'} balance={100000} size='xs' />
+                                </StaticCurrencySwitcherContainer>
+                            )}
+                        </div>
+                        <div
+                            className='static-dashboard-wrapper__body'
+                            style={has_applauncher_account && !isMobile() ? { height: '4rem' } : {}}
+                        >
+                            {!has_applauncher_account && (
+                                <StaticCFDAccountManager
+                                    type='all'
+                                    platform='options'
+                                    appname={is_eu_account_title}
+                                    description={`Get a real ${is_eu_title} account, start trading and manage your funds.`}
+                                    currency={currency}
+                                    has_account={has_account}
+                                    is_blurry={is_blurry}
+                                    is_onboarding_animated={is_onboarding_animated}
+                                    is_eu_user={is_eu_user}
+                                />
+                            )}
+                            {isMobile() && <Divider />}
+                        </div>
+
+                        <div
+                            className={classNames('static-dashboard-wrapper__body--apps', {
+                                'static-dashboard-wrapper__body--apps--eu': is_eu_user,
+                            })}
+                        >
+                            <div className={'static-dashboard-wrapper__body--apps-item'}>
+                                <StaticTradingAppCard
```

Comment:
Isn't it better to define an array for these StaticTradingAppCard contents and map through all items oa array instead  of repeating hardcoded component?

Reply:
You're right.
Since this component had a lot of changes recently, its code needs refactoring and I will do so after the release of our project.
I think it'd be better to just push this PR forward and spend time for clean ups after the release.